Creating-rosetta-terminal.md

```diff
@@ -0,0 +1,17 @@
+## How to create a Rosetta 2 terminal on Mac M1 computers
+
+#### Install Rosetta 2
+Rosetta 2 is only available for Mac computers with the M1 chip. If Rosetta 2 is not installed, prompt a rosetta 2 installation using the command:
+
+`softwareupdate --install-rosetta`
+
+#### Create a Rosetta terminal
+
+1. Open Finder and navigate to Application/Utilities. 
+2. Right-click the "Terminal" application and select "Duplicate". 
```

Comment:
This step seems very non-standard. I'm checking with some devs who use M1 macs to confirm that this is safe

Reply:
I haven't been able to find anyone else who has a good understanding of the potential side-effects of this, so let's add a disclaimer that this step might be dangerous and provide an alternative for users who don't want to take that risk

Reply:
Sure. Alternatively, we skip the "Duplicate" part and only instruct users to create a "session" terminal by clicking "open using Rosetta".